docs/workflow.md

```diff
@@ -31,3 +31,16 @@ To simulate Axe for Android (running on port 9051) for local testing of the app,
 ```sh
 yarn with:mock-axe-android start:electron:dev
 ```
+
+## Pull Request title tagging convention (Labelling commit messages)
+
+For every Pull Request, we prefer to add a tag in the PR title. This helps us in maintaining a good commit history as well as document better
+when creating Release notes.
+The convention that we follow is inspired from [SemVer](https://semver.org/) convention and is following:
+
+| Tag        | What it conveys                                                                                                                              |
```

Comment:
The spacing between the pipe characters doesn't actually impact the markdown. Some people like to keep it formatted as plaintext, others don't. AIWin uses the simplified style, but either one will work equally well

Reply:
if you look at the preview of this markdown, this is gonna space up to form a proper table. Also, spacing was done by prettier automatically, so i didn't touch it further 😄